rest/data/perf.go

```diff
@@ -265,6 +265,30 @@ func (dbc *DBConnector) ScheduleSignalProcessingRecalculateJobs(ctx context.Cont
 	return catcher.Resolve()
 }
 
+type ChangePointStub struct {
+	PerfResultID string `bson:"perf_result_id" json:"perf_result_id" yaml:"perf_result_id"`
+	Measurement  string `bson:"measurement" json:"measurement" yaml:"measurement"`
+}
+
+func (dbc *DBConnector) TriageChangePoints(ctx context.Context, changePoints map[string]string, status string) error {
+	ts := model.TriageStatus(status)
+	err := ts.Validate()
+	if err != nil {
+		return gimlet.ErrorResponse{
+			StatusCode: http.StatusUnprocessableEntity,
+			Message:    fmt.Sprintf("Invalid triage status: %s", status),
+		}
+	}
+	err = model.TriageChangePoints(ctx, dbc.env, changePoints, ts)
+	if err != nil {
+		return gimlet.ErrorResponse{
+			StatusCode: http.StatusInternalServerError,
+			Message:    "Could not triage change points",
```

Comment:
do you want this message to annotate/log something about the cause of the error passed?

Reply:
Updated to the following:
* If one of the results was not found for update, tell client which one w/ 404 response so they can fixup request; don't pollute logs with client request errors
* If this is not client error, log it, and only give generic information to client under 500 status code (don't leak implementation details)

Reply:
Only thing that I find a little janky is doing strings.contains on the error message. Surely there's a better way?